docs/ruleset_functions/get_proposed_hvac_modeled_with_virtual_cooling.md

```diff
@@ -0,0 +1,40 @@
+# get_proposed_hvac_modeled_with_virtual_cooling
+
+**Description:** Get the list of HeatingVentilationAirAconditioningSystem in which Appendix G Table G3.1 #10 d is applicable.
+
+**Inputs:**
+- **U-RMR**: The U-RMR to determine if an HVAC system has been designed or modeled as existing and with cooling.
+- **P-RMR**: To determine if the same HVAC system has been modeled with cooling in the P-RMR.
+
+**Returns:**
+- **proposed_hvac_modeled_with_virtual_cooling_list_p**: A list that saves all the HVAC systems in which Appendix G Table G3.1 #10 d are applicable (i.e. modeled with virtual cooling in the proposed).
+ 
```

Comment:
@claperle and @mkarpman is 'virtual cooling' a recognized word? I dont have a suggestion for another term, just wondering if that is what is used in the industry